keyboards/zos/65s/info.json

```diff
@@ -0,0 +1,100 @@
+{
+    "manufacturer": "ZOS",
+    "keyboard_name": "ZOS 65s",
+    "maintainer": "Zenn",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["D2", "D1", "D0", "D7", "B4", "B5", "F0", "B6", "C6", "F1", "C7", "F4", "F5", "F6", "F7"],
+        "rows": ["D5", "D4", "D6", "D3", "B7"]
+    },
+    "processor": "atmega32u4",
+    "url": "https://www.zoskb.com",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0001",
+        "vid": "0x25D5"
+    },
+    "layouts": {
+        "LAYOUT": {
```

Comment:
```suggestion
    "community_layouts": ["65_ansi_blocker"],
    "layouts": {
        "LAYOUT_65_ansi_blocker": {
```

Reply:
Thanks i've updated the layout name to LAYOUT_65_ansi. Also fixed an issue with the matrix, there was an error preventing it from compiling and qmk used the keyboard.h file instead.

Now that this is fixed, I've also deleted the keyboard.h file in the latest commit